lib/renuo/bin-check/command_result.rb

```diff
@@ -0,0 +1,10 @@
+module RenuoBinCheck
+  class CommandResult
+    attr_accessor :output, :error_output, :exit_code
```

Comment:
yes, you're right. I made an accessor, because i thought Factory Girl needs it, but it doesn't in this case :)